notebooks/10_bayesian_networks_sampling/index.md

```diff
@@ -0,0 +1,233 @@
+<div align="center">
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+    <h1 style="font-size: 40px; margin: 10px 0;">AI - Inference in Bayesian Networks: Sampling</h1>
+    <h1 style="font-size: 20px; font-weight: 400;">Sharif University of Technology - Computer Engineering Department</h1>
+    <br>
+    <h4 style="font-size: 18px; font-weight: 400; color:#555">Alireza Honarvar, Navid Eslami, Ali Najibi</h4>
+    <br>
+    <br>
+    <br>
+    <br>
+    <br>
+</div>
+<hr>
+
+Table of Contents
+==============
+
+- [Table of Contents](#table-of-contents)
+- [Introduction](#introduction)
+- [Basic Idea](#basic-idea)
+- [Sampling from Given Distribution](#sampling-from-given-distribution)
+- [Prior Sampling](#prior-sampling)
```

Comment:
This bullet and following three other kinds of sampling may look more structured if you add a bullet named "Sampling in Bayes' Nets" and add these as sub-bullets

Reply:
Thank you for your notice. It has been resolved.